include/aspect/introspection.h

```diff
@@ -293,6 +293,13 @@ namespace aspect
        */
       IndexSets index_sets;
 
+      enum AdvectionMethod
```

Comment:
In many places we wrap the enum in a struct an call the enum itself ``kind`` to scope the enum values.